packages/@react-native-windows/codegen/src/generators/ParamTypes.ts

```diff
@@ -122,6 +161,9 @@ function translateParam(
     }
     case 'TypeAliasTypeAnnotation':
       return decorateType(getAliasCppName(param.name), target);
+    case 'EnumDeclaration':
```

Comment:
Will these types come from the rn codegen schema used in the 0.71 version of rn-codegen?

Reply:
This version of the `@rnw/codegen` still uses `react-native-typescript-tscodegen`, so it doens't really matter about the `@rn/codegen` version.